shell.nix

```diff
@@ -36,7 +36,7 @@ let
 
     # Prevents cabal from choosing alternate plans, so that
     # *all* dependencies are provided by Nix.
-    exactDeps = true;
+    exactDeps = false;
```

Comment:
Is this change supposed to be a part of this PR? I don't really know anything about nix, just making sure that this wasn't a mistake.